ql/src/java/org/apache/hadoop/hive/ql/plan/FileSinkDesc.java

```diff
@@ -390,6 +394,17 @@ public boolean isCompactionTable() {
         : AcidUtils.isCompactionTable(getTableInfo().getProperties());
   }
 
+  /**
+   * @return true if the compaction type is 'REBALANCE', false otherwise.
+   */
+  public boolean isRebalanceCompactionTable() {
```

Comment:
I think that should be in Util class

Reply:
this method should take a Properties object as input

Reply:
To move it into AcidUtils it should be sp;itted to two methods like the isMajorCompaction() to handle both Properties and Map. Considering it has a single usage I thought it's not worth moving this method there.